packages/angular/projects/ui-angular/src/lib/directives/proxies.ts

```diff
@@ -0,0 +1,1468 @@
+/* tslint:disable */
```

Comment:
As we discussed earlier, it might be possible to export this file directly from `@aws-amplify/ui-components` with something like [this](https://github.com/aws-amplify/amplify-js/compare/export-proxies-from-ui-components). That way we don't have to manually copy this over. Not immediate, but let's try this when we get the time!

Reply:
Sure! OK, let's look at a separate PR to do that. Since it would take updates to the `@aws-amplify/ui-components`